Fix things

subliun opened the pull request.

LittleVulpix committed:
Update copyright year/link

LittleVulpix committed:
Update addkeyweb_success.html

LittleVulpix committed:
Update api_error_pretty.html

LittleVulpix committed:
Update edit_ui.html

LittleVulpix committed:
Update fourohfour.html

LittleVulpix committed:
Update lookup_home.html

LittleVulpix committed:
Update onemomentplease.html

LittleVulpix committed:
Update public_userlist.html

LittleVulpix committed:
Merge pull request #1 from LittleVulpix/Update-copyright

Update copyright + link for the web UI

LittleVulpix committed:
Merge pull request #2 from subliun/master

make things that were worse before better in the hopes of unbreaking …

SkyzohKey commented:
Wow I known ToxMe code was shit but not at this point. Could have been using templates, at least. e.e

LittleVulpix committed:
Get rid of some crawlers

LittleVulpix committed:
Add the robots.txt file to be served

as a static file

LittleVulpix committed:
Merge pull request #3 from LittleVulpix/LittleVulpix-robotstxt

Little vulpix robotstxt

LittleVulpix committed:
Also delete item from cache...

... to prevent nonexistant records from showing up and blocking addition of a user.

LittleVulpix committed:
Merge #4 - Fix cache for user deletion

- Prevent already deleted items from showing up in results (fixes inability to delete and re-add the same name without the need to restart toxme)

LittleVulpix committed:
Switch original link.

LittleVulpix committed:
Disallow crawling of specific users, allow root

Only allow root to be crawled.

LittleVulpix committed:
Add a way to contact me

This is mostly for people who have some kind of request related to having their old account removed after they forgot the password.

LittleVulpix committed:
Add a more apparent way of contacting me

This is for people who want to change their account but don't remember their password.

Note that you will always need to be able to prove the account you want to modify is yours. In case of losing the toxme.io password, this means you must be able to log into this account and add me from it.

LittleVulpix committed:
Merge pull request #5 from LittleVulpix/Contact-the-maintainer

Add a more apparent way of contacting me

LittleVulpix committed:
Modify the way of contacting me, add tox contact

SkyzohKey commented on static/robots.txt:
```diff
@@ -0,0 +1,3 @@
+User-agent: *
+Allow: /$
```
I'm pretty sure the `$` mark is only useful in the router regex, marking the end of the line.

bennabiy committed:
Update README.md

pip3 update in multiple locations

LittleVulpix committed:
Merge pull request #7 from bennabiy/patch-2

Update README.md

LittleVulpix committed:
Update README.md

libsodium is no longer available for direct download so we compile it manually.

LittleVulpix committed:
Merge pull request #8 from LittleVulpix/fix-missing-dependency

Update README.md to reflect missing dependency

LittleVulpix committed:
@toxme.io is not an e-mail domain

Make it more apparent because people were apparently confused by it.

LittleVulpix committed:
Merge pull request #9 from LittleVulpix/not-an-email

@toxme.io is not an e-mail domain

TheDcoder committed:
Fix headings in README

LittleVulpix committed:
Merge pull request #10 from TheDcoder/patch-1

Fix headings in README

LittleVulpix committed:
Show full tox ID even when user has no bio

Previously, full tox id shows only when user had bio. Makes no sense to do it this way.

LittleVulpix committed:
Merge pull request #11 from LittleVulpix/LittleVulpix-show-id

Show full tox ID even when user has no bio

LittleVulpix committed:
Add echobot as a test contact

Sick and tired of people adding me to "test toxme". I'm here for technical issues.

LittleVulpix committed:
Merge pull request #12 from LittleVulpix/LittleVulpix-echobot

Add echobot as a test contact instead of myself.

LittleVulpix committed:
Add favicon to the template

No more blank favicon on the pain page of toxme.

LittleVulpix committed:
Upload the favicon file

looking glass+tox padlock. I think it fits.

LittleVulpix committed:
Merge pull request #13 from LittleVulpix/favicon

Add favicon to the template

robinlinden committed:
Fix headings in doc/config.md

LittleVulpix committed:
Merge pull request #15 from robinlinden/fix-markdown

Fix headings in doc/config.md

Commit:
Omit Bios from public view (#16)

squashed changes to remove reference to bios from public view

* Omit Bios from public view

Hello darkness my ol' friend.

* more fixes

@LittleVulpix, squash this so I don't have to git clone

LittleVulpix committed:
toxme.io copyright update for 2018 (#17)

* add_ui year bump

* addkeyweb_success year bump

* api_error_pretty year bump

* edit_ui year bump

* fourohfour year bump

* lookup_home year bump

* onemomentplease year bump

* public_userlist year bump

LittleVulpix committed:
Remove CDN dependency for the font+css (#18)

Get rid of CDN dependency

LittleVulpix committed:
This is why we can't have nice things. (#19)

People constantly add me "no I just want to test toxme". The link to echobot for "TESTING TOXME" is literally right above mine! So there, no direct contact easily visibly on the site now. Enjoy your echobot.

SkyzohKey commented:
About the latest change, maybe make a support@toxme.io or admin@toxme.io address

LittleVulpix committed:
No more friends (#22)

* Remove "not in use" line.

* Update logic to disable only friend listing

When cfg value is set to 0, it now still allows adding/editing but no more public list of any user; you can still query normally however.

LittleVulpix committed:
Don't redirect to friends, friends are no more.

Could be done with the CFG but really, I don't wanna redirect to that page at all.

LittleVulpix committed:
It is 2019 my dudes. (#24)

All the files updated with new copyright year.

LittleVulpix committed:
RIP

https://blog.fefe.de/?ts=a3695410 and this is why we can't have nice things.
